packages/server/graphql/mutations/helpers/updateReflectionLocation/addReflectionToGroup.ts

```diff
@@ -63,7 +71,25 @@ const addReflectionToGroup = async (
     }).run()
 
     const nextTitle = getGroupSmartTitle(nextReflections)
-    await updateSmartGroupTitle(reflectionGroupId, nextTitle)
+    if (
+      reflectionGroup.title === reflectionGroup.smartTitle &&
+      nextReflections.length === 2 &&
```

Comment:
-1 why do we want to update the smart title only if the nextReflection length is 2? If the nextReflection group has 3 reflections but has a smart title, and the oldReflection group has a custom title, would we want to update the title of the new group?

Perhaps something like this would work?
```
    const newGroupHasCustomTitle = reflectionGroup.title !== nextTitle
    const oldGroupHasCustomTitle = oldReflectionGroup.title !== oldReflectionGroup.smartTitle

    if (!newGroupHasCustomTitle && oldGroupHasCustomTitle) {
```

Reply:
@nickoferrall here I only handle a case where you drag a single card with a title on another **single card** with no visible title.

`nextReflections.length === 2` is a bit confusing, as when we do this check the dragged card is already added to the group, so length `2` includes the dragged card.

But I think it make sense to override any smart title if you drag a card with a custom title over it.

I updated the code and changed the logic